File: app/src/main/java/org/techtown/find_gas_station/Data/kakaoResponseModel/kakao/DirectionResponse.java
package org.techtown.find_gas_station.Data.kakaoResponseModel.kakao;

import com.google.gson.annotations.SerializedName;

public class DirectionResponse {


    private String trans_id;


    private Route[] routes;

    public String getTrans_id() {
        return trans_id;
    }

    public void setTrans_id(String trans_id) {
        this.trans_id = trans_id;
    }

    public Route[] getRoutes() {
        return routes;
    }

    public void setRoutes(Route[] routes) {
        this.routes = routes;
    }
}

File: app/src/main/java/org/techtown/find_gas_station/Data/kakao/Response/DirectionResponse.kt
package org.techtown.find_gas_station.Data.kakao.Response

import com.google.gson.annotations.SerializedName

data class DirectionResponse(

    @SerializedName("trans_id")
    var transId : String,

    @SerializedName("routes")
    var routes : List<Route>

)